resources/js/remark-plugins/disable-constructs.ts

```diff
@@ -76,6 +76,7 @@ const disabled: Record<DisabledType, Construct[]> = {
     'autolink',
     'labelEnd',
     'labelStartLink',
+    'lineEnding',
```

Comment:
remove from "all disabled list" instead? If things are broken with this disabled